modin/experimental/backends/omnisci/query_compiler.py

```diff
@@ -23,6 +24,59 @@
 from pandas.core.dtypes.common import is_list_like
 
 
+def is_default_frame(value):
```

Comment:
I'd suggest `inoperable` instead of `default` for naming.

Reply:
done